tests/foreman/longrun/test_oscap.py

```diff
@@ -252,7 +254,7 @@ def test_positive_upload_to_satellite(self):
                     )
                     self.hosts.update(
                         name=vm._target_image,
-                        domain_name=vm._domain,
+                        domain_name=vm.domain,
```

Comment:
could we make this change in the context of a work-around of that bug to skip the rhel6 oscap case?
any way think should be done in pair with vm._target_image that access  a private attribute
better way 

```python 
host_name, _, domain_name = vm.host_name.partion('.')
self.hosts.update(name=host_name,  domain_name=domain_name ...
```

Reply:
The bug , is only for the default rhel6 content , we can use a workaround where we can upload a older rhel6 custom content . Will look into workaround